packages/apps-v5/src/commands/keys/add.js

```diff
@@ -40,7 +40,7 @@ async function run (context, heroku) {
   let findKey = async function () {
     const defaultKey = path.join(sshdir, 'id_rsa.pub')
     if (!(await fs.pathExists(defaultKey))) {
-      cli.console.error('Could not find an existing SSH key at ~/.ssh/id_rsa.pub')
+      cli.console.error('Could not find an existing SSH key at ' + path.join('~', '.ssh', 'id_rsa.pub'))
```

Comment:
Use string interpolations over concatenation. This preserves a style choice across the cli.

Reply:
I think in this particular case I'm fine with concatenation, (where it's just 1 concatenation) but in most of the other ones in this PR I agree